makefile

```diff
@@ -13,59 +13,39 @@ help:
 	@echo Iceberg Local Development Environment
 	@echo -------------------------------------------------------------------------
 	@echo install   : Install the required packages
+	@echo run       : Run the Iceberg backend
 	@echo start     : Build and run the Iceberg container
 	@echo stop      : Stop and remove the Iceberg container and image
-	@echo restart   : Restart the Iceberg container (without rebuilding image)
-	@echo exec      : Open a shell inside the Iceberg container
 	@echo logs      : Show the logs of the Iceberg container
-	@echo sc        : Show the contents of the settings.json file inside the Iceberg container
-	@echo ec        : Edit the settings.json file inside the Iceberg container
-	@echo update    : Update this repository from GitHub and rebuild image
-	@echo frontend  : Start the frontend development server
-	@echo backend   : Start the backend development server
+	@echo lint      : Run the linter and type checker
+	@echo test      : Run the tests
+	@echo pr-ready  : Run the linter and tests
 	@echo -------------------------------------------------------------------------
 
 install:
-	@python3 -m pip install -r requirements.txt --break-system-packages
+	@pip install -r requirements.txt --upgrade --break-system-packages
```

Comment:
possibly switch to using poetry install? would handle packages better than this :/